tfjs-backend-webgpu/src/shader_preprocessor.ts

```diff
@@ -597,10 +596,7 @@ export function generateGetOutputCoords(
   const outRank = outShape.length;
   if (x.length === outRank) {
     const dtype = getCoordsDataType(outRank);
-    const snippet =
-        `fn getOutputCoordsWithFlatDispatchLayout(globalId : vec3<u32>, localId : vec3<u32>, numWorkgroups: vec3<u32>) -> ${
-            dtype}{
-      let globalIndex = getGlobalIndex(globalId, localId, numWorkgroups);
+    const snippet = `fn getOutputCoords(globalIndex: i32) -> ${dtype}{
```

Comment:
I prefer `fn getOutputCoords()` instead of `fn getOutputCoords(globalIndex: i32)`

Reply:
Done